server/lib/project/projectManager.js

```diff
@@ -164,6 +167,25 @@ function* acceptBid(admin, contract, buyer, bidId, name) {   // FIXME should go
   return result;
 }
 
+// throws: ErrorCodes
+function* rejectProject(admin, contract, buyer, name) {
+  rest.verbose('rejectProject', {admin, buyer, name});
```

Comment:
please model this after `settleProject`.
follow the settleProject path and match every aspect of it